tests/crypto_test.cpp

```diff
@@ -332,6 +332,19 @@ TEST_F(CryptoTest, sortition_rate) {
   EXPECT_EQ(count, valid_sortition_players * round);
 }
 
+TEST_F(CryptoTest, binomial_distribution) {
```

Comment:
this test does not test anything

Reply:
I like your test. I modify it a litter bit. It could test both cases with "total count < committee size" and "total count >= committee size". The test could test if the getBinominalDistribution always return same value with introduce binary search.
If you don't mind, I prefer to keep the test here.

Reply:
I just do no understand when `getBinominalDistribution` with same params could return different result